sui-execution/src/lib.rs

```diff
@@ -15,11 +15,13 @@ pub mod executor;
 pub mod verifier;
 
 mod latest;
+mod suivm;
 mod v0;
 
 #[cfg(test)]
 mod tests;
 
+pub const SUIVM: u64 = u64::MAX;
```

Comment:
Note the funky version allocation (it's not a problem, just calling attention to it) -- this is because it's a feature branch, and not a version snapshot.

Reply:
oh wait I will remove my second upload so I can commit the first change - lol